### ISSUE TITLE
Create UI for JWT debuggin/decoding

### ISSUE THREAD

**Issue by Chris2011** [javierllorente/netbeans-rest-client#17]
If you know https://jwt.io/ I had an idea how cool will this, if we can add this in this Rest Client? I see this really here in this plugin, because its also about auth, etc. So what do you think? I will create this UI soon and send a PR. You can assign this ticket to me.

**Comment by javierllorente** [comment]
How could that be integrated with the current UI? We could add a new option under Authorisation tab -> dropdown menu. The upper tab body and the bottom one could be reused for the JWT. We would need a new widget for verifying the signature. What else?

**Comment by Chris2011** [comment]
Yes, this will be an option not directly inside the current UI but a new tab/subtab or other area implementing, when the user click something. It is similar to the postman stuff where you can use / see curl commands or others but not by default. We can have a toolbar which opens new stuff: https://learning.postman.com/docs/getting-started/importing-and-exporting/importing-curl-commands/#convert-a-postman-request-to-a-curl-command

**Comment by Chris2011** [comment]
ATM I need to check the JWT.io debugger what is needed her vor but yes a new dialog or whatever, with a input panel and 3 outputs which shows the result.

**Comment by javierllorente** [comment]
I have just seen that in Postman there is "JWT bearer" under "Authorization". Perhaps you could get some inspiration from there. 
I think that this feature would be very useful. Thanks!

**Comment by Chris2011** [comment]
Thx for the hint, will have a look.
